web/packages/core/src/ruffle-player.ts

```diff
@@ -1588,37 +1588,40 @@ export class RufflePlayer extends HTMLElement {
      * @returns A string containing all the data included in the panic.
      */
     private getPanicData(): string {
-        const dataArray = [];
```

Comment:
(Nit/comment: I imagine that the original thinking behind creating an array of strings and then joining them at the end might be to avoid the cost of repeated string concatenation. But, I believe that the number and length of the string values here are limited, and that `getPanicData()` is a method that is called very infrequently if at all, so the performance of the string concatenation here does not really matter and I think the changes makes the code cleaner).

Reply:
No, this was done to prevent issues when reporting a bug (#2318). Some parts later moved to the new function `getPanicData` (#7577).

Reply:
Huh. I think I would have just made an interface like:

```typescript
interface ErrorData {
    errorText: string;
    stackIndex: number;
}
```

or something, instead of adding `stackIndex` to an array. But I have only skimmed through the code, and this might be way off.

But I would guess that this means that this part of the PR (still) looks good, I believe.